src/Cake.Tfs.Tests/PullRequest/TfsPullRequestTests.cs

```diff
@@ -32,13 +32,30 @@ public void Should_Throw_If_Settings_Are_Null()
                 // Given
                 var log = new FakeLog();
                 TfsPullRequestSettings settings = null;
+                var gitClientFactory = new FakeGitClientFactory();
 
                 // When
-                var result = Record.Exception(() => new TfsPullRequest(log, settings));
+                var result = Record.Exception(() => new TfsPullRequest(log, settings, gitClientFactory));
 
                 // Then
                 result.IsArgumentNullException("settings");
             }
+
+            [Fact]
+            public void Should_Return_Valid_Pull_Request()
+            {
+                // Given
+                var log = new FakeLog();
+                var settings = new TfsPullRequestSettings(
+                    new Uri("http://MyServer/tfs/MyCollection/MyTeamProject/_git/MyRepoName"), 42, new TfsNtlmCredentials());
+                var gitClientFactory = new FakeGitClientFactory();
```

Comment:
To simplify writing of test cases we could create a fixture class, which does all this bootstrapping code to some default values, which can be overwriten as required by the specific test case (from the principle like done [here](https://github.com/cake-contrib/Cake.Issues.PullRequests/blob/develop/src/Cake.Issues.PullRequests.Tests/PullRequestsFixture.cs))